python_modules/libraries/dagster-datadog/dagster_datadog/resources.py

```diff
@@ -0,0 +1,59 @@
+from datadog import initialize, statsd
+
+from dagster import Dict, Field, ResourceDefinition, String
+
+
+class DataDogResource:
+    def __init__(self, api_key, app_key):
+        self.api_key = api_key
+        self.app_key = app_key
+        initialize(api_key=api_key, app_key=app_key)
```

Comment:
wow this is very global state-y/side-effect-y

Reply:
yeah, this lib is gross - sets these top-level globals:

![image](https://user-images.githubusercontent.com/609349/56540887-e06a5500-651e-11e9-8bb7-2342d9b9a52d.png)

Reply:
uffda